src/coreclr/jit/gentree.cpp

```diff
@@ -9961,81 +9961,7 @@ void Compiler::gtUpdateNodeSideEffects(GenTree* tree)
 
 bool GenTree::gtSetFlags() const
 {
-    //
-    // When FEATURE_SET_FLAGS (TARGET_ARM) is active the method returns true
-    //    when the gtFlags has the flag GTF_SET_FLAGS set
-    // otherwise the architecture will be have instructions that typically set
-    //    the flags and this method will return true.
-    //
-    //    Exceptions: GT_IND (load/store) is not allowed to set the flags
-    //                and on XARCH the GT_MUL/GT_DIV and all overflow instructions
-    //                do not set the condition flags
```

Comment:
I don't see any `GT_IND` flag that overlaps `GTF_SET_FLAGS``  today, so I've removed this special casing for `GT_IND`.

I think as a separate clean up we can remove `GTF_SET_FLAGS` into LIR flags.